apps/web/src/lib/rehypePrettyCode.ts

```diff
@@ -0,0 +1,21 @@
+import type { Options } from "rehype-pretty-code";

```

Comment:
Can we rename the file to `rehype-pretty-code` to stick to the file name convention.